core/Tracker/Cache.php

```diff
@@ -105,6 +107,8 @@ static function getCacheWebsiteAttributes($idSite)
             $cache->save($cacheId, $content, self::getTtl());
         }
 
+        Tracker::restoreTrackerPlugins();
```

Comment:
The `Tracker\Cache` loads core plugins when in tracker mode, we need to restore them after the cache has been initialized